pkg/internal/apis/certmanager/validation/certificate_test.go

```diff
@@ -125,7 +125,8 @@ func TestValidateCertificate(t *testing.T) {
 				},
 			},
 			errs: []*field.Error{
-				field.Required(fldPath.Child("commonName", "dnsNames", "uriSANs"), "at least one of commonName, dnsNames, or uriSANs must be set"),
+				field.Required(fldPath.Child("commonName", "dnsNames", "uriSANs", "emailSANs"),
```

Comment:
similar to above., this is not correct (although not introduced by you!)

Reply:
above? sorry GitHub sometimes mess up the review order

Reply:
found it!